wpeverest/user-registration
# Dev - Compatibility for Image based Captcha

### All Submissions:



### Changes proposed in this Pull Request:

Previously, captcha field had only two format (Math and QA). This PR introduces new format Image that uses icon group for captcha on form.
Pro Dependency: https://github.com/wpeverest/user-registration-pro/pull/400
Closes # .

### How to test the changes in this Pull Request:

1. Drag a captcha field, select image on captcha format.
2. Choose icons and Change form settings and icon group.
3. Verify by submitting form.

### Types of changes:



### Other information:




### Changelog entry

> Dev - Compatibility for Image based Captcha
